doc: Instructions to update the docker grpc image

papeldeorigami opened the pull request:
Add information on how to update the docker image that create the local protocol buffers.

papeldeorigami committed:
doc: Instructions to update the docker grpc image

aapeliv reviewed (dismissed):
Ah, thank you!

darrenvong commented on app/readme.md:
````diff
@@ -54,6 +54,17 @@ If you are on **Windows** (without WSL2), run the following command:
 docker run --rm -w /app -v %cd%:/app registry.gitlab.com/couchers/grpc sh -c "cat generate_protos.sh | dos2unix | sh"
 ```
 
+### How to update the protocol buffers
````
```suggestion
### How to update the protocol buffers compiler
```
Very minor nitpick as the above steps already update the protocol buffers, but errors because of an updated compiler.

darrenvong replied:
In fact, I'll just commit the suggestion to save more round trips of this being blocked from merging :p

darrenvong replied:
```suggestion
### How to update the protocol buffers compiler
```

darrenvong committed:
Tweak title on updating protobuf compiler

aapeliv reviewed (dismissed):
Actually, I think you can get around this by using the `--pull always` flag on the `docker run` command; like this:

```sh
docker run --pull always --rm -w /app -v $(pwd):/app registry.gitlab.com/couchers/grpc ./generate_protos.sh
```

papeldeorigami committed:
Instructions to pull always the latest grpc docker

Add the flag `--pull always` to the grpc docker command in the readme instructions

darrenvong committed:
Tweak title
